deploy/helm/kubecf/assets/operations/instance_groups/bits.yaml

```diff
@@ -50,13 +50,12 @@
   value: local
 
 # Attach a persistent disk to bits-service VM to store eirinifs
-# TODO: storage class type should be configurable
-# - type: replace
-#   path: /instance_groups/name=bits/persistent_disk_type?
-#   value: ???
 - type: replace
   path: /instance_groups/name=bits/persistent_disk?
   value: 20480 # 20GB
+- type: replace
+  path: /instance_groups/name=bits/persistent_disk_type?
```

Comment:
Since the path exists in the original manifest, there is no need to use `?`. I believe it's true for all the cases in this PR.

I think it also makes sense to use a conditional render for this operation to exist only if `kube.storage_class` is set. The reason behind it is that we don't change the current behaviour of using whatever default storage class the user has.

Also, we need to add this key to the default `values.yaml`. I think there is a good place to document it. The default value can simply be null `~`.

Reply:
I think `?` is still needed for bits (https://github.com/cloudfoundry/cf-deployment/blob/master/operations/bits-service/use-bits-service.yml#L86). I will remove from the rest. Adding conditional and default value with docs.

Reply:
Is this the default `values.yaml` file you mean? https://github.com/SUSE/kubecf/blob/master/deploy/helm/kubecf/values.yaml . Adding there.

Reply:
That's the one!